lib/packs/private.rb

```diff
@@ -550,7 +553,7 @@ def self.get_info(packs: Packs.all, format: :detail, types: %i[privacy dependenc
           pack_name: pack.name,
           owner: owner.nil? ? 'No one' : owner.name,
           size: pack.relative_path.glob('**/*.rb').count,
-          public_api: pack.relative_path.join('app/public')
+          public_api: pack.relative_path.join(package_by_name[pack.name].public_path)
```

Comment:
I think I'm missing something here. Why create a hash and get the pack from the hash rather than calling `pack.public_path`?

Reply:
Nevermind, I see that it's a `ParsePackwerk::Pack` and not a `Packs::Pack`.